dell-restore-system-dhc

```diff
@@ -0,0 +1,89 @@
+#! /usr/bin/python3
```

Comment:
However since this file is mostly identical to the original, what do you think about just changing the behavior of the original to add support for a boolean flag that defaults to false like `--dhc`.

Reply:
I remove dell-restore-system-dhc file, and add a -d option for dhc in dell-restore-system, meanwhile I also add judgement for -d option, because only some platform need it.